NICE建設LT8-商談・工事管理/src/org/desktop/js/kouji-3.5.0.js

```diff
@@ -16,10 +16,376 @@
  * @version 3.5.0
  * @customer （2023-06-25）
 */
-(function (config, sncLib) {
+jQuery.noConflict();
+(function ($, config, sncLib) {
     'use strict';
     const cfgKouji = config.kouji;
     const cfgKoujiFields = config.kouji.fields;
+    let nonEditFields = null;
+    let execNum = 0;
+    let koujiDataPerPage = null;
+
+    // 着工前物件
+    const shinchokuKanriView = [
```

Comment:
Thay đổi lại cấu trúc của config phần subtable, thay vì chỉ định từng trường trong 1 subtable thì nên chia thành 2 mảng:
1.Mảng include: chỉ cần chỉ định các trường muốn hiển thị
2.Mảng exclude: chỉ cần chỉ định các trường không muốn hiển thị
3.Trường hợp cả 2 mảng đều trống(tức ko config chỉ định trường nào) -> mặc định get toàn bộ các trường của subtable đc chỉ định.
4.Trường hợp có nhập dữ liệu cho cả 2 mảng -> hiển thị thông báo lỗi yêu cầu chỉ đc thiết lập 1 trong 2 mảng

Reply:
Em đã xử lý cho mảng includes các trường của subtable sẽ được hiển thị như đã thảo luận trong Slack ạ